Makefile

```diff
@@ -154,6 +159,12 @@ terraform-monitoring-plan: terraform-monitoring-init ## make passcode=MyPasscode
 terraform-monitoring-apply: terraform-monitoring-init ## make passcode=MyPasscode terraform-monitoring-apply
 		cd terraform/monitoring && terraform apply -input=false -auto-approve
 
+##@ rails console. Ability to rail console to apps on PaaS
+
+console: ## make qa console
```

Comment:
Could you add this to the ssh section of hosting.md too please?

Reply:
sure....